File: Daily Test/C/Reverse Odd Columns 3.4.2026.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int mat[n][n];

    for(int i = 0; i < n; i++)
        for(int j = 0; j < n; j++)
            scanf("%d", &mat[i][j]);

    for(int i = 0; i < n; i++){
        for(int j = 0; j < n; j++){
            if(j % 2 == 0)
                printf("%d ", mat[n-1-i][j]);
            else
                printf("%d ", mat[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Daily Test/Python/Reverse Odd Columns 3.4.2026.py
n = int(input())
mat = [list(map(int, input().split())) for _ in range(n)]

for i in range(n):
    for j in range(n):
        if j % 2 == 0:
            print(mat[n-1-i][j], end=" ")
        else:
            print(mat[i][j], end=" ")
    print()